cibyl/sources/elasticsearch/api.py

```diff
@@ -47,7 +47,7 @@ def __init__(self: object, driver: str = 'elasticsearch',
                       from exception
             self.es_client = ElasticSearchClient(host, port).connect()
 
-    def get_jobs(self: object, **kwargs) -> list:
+    def get_jobs(self: object, **kwargs: str) -> list:
```

Comment:
The type annotation here should be dict, not str, shouldn't it?

Reply:
In this case we're receiving dictionaries with string keys, shouldn't it be **string** in this case? Reference: https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values

Maybe I'm confusing with this. Let me know your opinion, please.

Reply:
I didn't know that, so you're right it should not be dict. However, according that PEP, the type annotated is the value, not the keys. So it should be Argument, not str.

Reply:
It makes sense. I’ll do the change later. Thanks!

Reply:
Changed.